setup.cfg

```diff
@@ -23,18 +24,19 @@ classifiers =
 [options]
 include_package_data = True
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.7
 zip_safe = False
 install_requires =
     click-default-group>=1.2.2,<2.0.0
     Flask-WTF>=0.15.1
     fs>=2.0.10,<3.0
-    invenio-accounts>=1.4.10
 
 [options.extras_require]
 tests =
+    pytest-black>=0.3.0
     future>=0.18.2
     invenio-access>=1.4.0
+    invenio-accounts>=1.4.10
```

Comment:
why is it necessary to add the new dependency ?

Reply:
You are right. It is not necessary. It will be installed anyway. My thinking was to explicitly show which packages are used in the tests.

Reply:
the problem with that approach is that we would have to maintain the minimal version in several places, instead of one

Reply:
removed